smartsim/_core/launcher/dragon/dragonConnector.py

```diff
@@ -245,9 +245,11 @@ def load_persisted_env(self) -> t.Dict[str, str]:
 
         with open(config.dragon_dotenv, encoding="utf-8") as dot_env:
             for kvp in dot_env.readlines():
-                split = kvp.strip().split("=", maxsplit=1)
-                key, value = split[0], split[-1]
-                self._env_vars[key] = value
+                # skip any commented lines
+                if not kvp.startswith("#"):
+                    split = kvp.strip().split("=", maxsplit=1)
+                    key, value = split[0], split[-1]
+                    self._env_vars[key] = value
```

Comment:
In the launchsettings, I remember we log a message if a value is overwritten, consider here - but also not sure if really needed...

if key in self._env_vars and key != self._env_vars[key]:
            logger.warning(f"Overwritting argument '{key}' with value '{value}'")

Reply:
it's used in a fresh process without the env vars. not possible to overwrite